homeassistant/components/camera/amcrest.py

```diff
@@ -4,21 +4,158 @@
 For more details about this platform, please refer to the documentation at
 https://home-assistant.io/components/camera.amcrest/
 """
+import asyncio
 import logging
+from requests import RequestException
+from urllib3.exceptions import ReadTimeoutError
 
-from homeassistant.components.amcrest import (
-    DATA_AMCREST, STREAM_SOURCE_LIST, TIMEOUT)
-from homeassistant.components.camera import Camera
+import voluptuous as vol
+
+from custom_components.amcrest import (
```

Comment:
This should be imported from `homeassistant.components.amcrest`

Reply:
Correct. In fact, this PR probably needs changes I made to homeassistant/components/amcrest.py as well, and possibly my new file which would go in homeassistant/components/binary_sensor/amcrest.py, too.